carbon-design-system/carbon-addons-iot-react
# style(composed-modal): fix close button RTL issues

Closes #2396 

**Summary**

- fixes RTL styles for composed modal

**Change List (commits, features, bugs, etc)**

- fixes composed modal close button position in RTL

**Acceptance Test (how to verify the PR)**

- open any composed modal story
- turn on RTL
- confirm close button is on the left side, not the right

**Regression Test (how to make sure this PR doesn't break old functionality)**

- n/a



**Things to look for during review**
